.travis.yml

```
@@ -26,7 +25,7 @@ stages:
   - name: test
     if: not (branch ~= /^(?i:feature).*$/)
   - name: build
-    if: type = push && (branch = master || branch = develop || branch ~= /^(?i:release|hotfix|feature).*$/)
+    if: type = push && (branch = master || branch = develop || branch ~= /^(?i:release|hotfix|feature).*$/) || $TRAVIS_COMMIT_MESSAGE == *"trigger build"*
```

```suggestion
    if: type = push && (branch = master || branch = develop || branch ~= /^(release|hotfix|feature).*$/) || $TRAVIS_COMMIT_MESSAGE == *"trigger build"*
```
?i: is not now for regex by me

---

Outdated by change at the base